### DISCUSSION TITLE
Got this error when i using this tool

### DISCUSSION THREAD

**Discussion by ZhangTomLiang** [originjs/webpack-to-vite#96] [Q&A]
Hellow,

When I make the conversion, I am prompted with this error.

node: v16.14.0

tomzhang@192 ruoyi-ui % sudo webpack-to-vite -d ./

Password:
******************* Webpack to Vite *******************
Project path: ./
progress [██████████████████████░░░░░░░░░░░░░░░░░░] 56% | AST Parsing: /Volumes/Application/project_station/RuoYi-Vue-master/ruoyi-ui/src/views/system/user/profile/userInfo6
Error: Cannot find module 'script-ext-html-webpack-plugin'
Require stack:
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Plugin.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Resolve.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Config.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/transform/transformVuecli.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/transform/transformer.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/generate/geneViteConfig.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/cli/cli.js
- /usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.toConfig (/usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Plugin.js:52:18)
    at /usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Config.js:129:61
    at Array.map (<anonymous>)
    at module.exports.toConfig (/usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Config.js:129:40)
    at setTransformParamsWithHtmlConfig (/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/ast-parse/astParse.js:151:70)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Plugin.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Resolve.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/node_modules/webpack-chain/src/Config.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/transform/transformVuecli.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/transform/transformer.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/generate/geneViteConfig.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/src/cli/cli.js',
    '/usr/local/lib/node_modules/@originjs/webpack-to-vite/dist/bin/index.js'
  ]
}

**Comment by Chieffo2021** [comment]
Are you using the latest version of webpack-to-vite? Please make sure you have installed the dependencies in your project.

**Comment by konpeki622** [comment]
Not support it now, please transform manually.
